### PR TITLE
Drop Sendable and Receivable traits

### PR DESCRIPTION
**Pull request by Ekleog**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,119 +1,19 @@
-extern crate libc;
```
Comment (nagisa) [inline src/lib.rs]: `extern crate` has to stay for `2015` edition crates, no?

**Comment by nagisa** [comment]
I think `Sendable` should stay, as it has none of the issues `Receivable` did.

File: src/lib.rs (hunk 1 above)
Comment (Ekleog) [reply src/lib.rs]: Hmm it looks like it compiles locally, despite the `edition = "2015"` in the `Cargo.toml`, so I'd guess it doesn't have to (though I have no idea to which version of rustc that ties us).

Do you want it back?

File: src/lib.rs [hunk 2]
```diff
@@ -423,10 +345,12 @@ mod tests {
     fn sending_nonraw() {
```
Comment (nagisa) [inline src/lib.rs]: This test can go.

**Comment by Ekleog** [comment]
My feeling around `Sendable` is that the inconsistency of sending `File` but being able to receive only `RawFd` is more damaging than the convenience gain of not having to write `.as_raw_fd()`

File: src/lib.rs (hunk 2 above)
Comment (Ekleog) [reply src/lib.rs]: Done

**Comment by nagisa** [comment]
r=me with `extern crate` back. I’m surprised edition 2015 works without it, but I guess it is not a breaking change to not require it.

**Comment by Ekleog** [comment]
Done, thanks!

**Pull request merged**